gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -430,6 +430,12 @@
   public static final String SOURCE_FILEBASED_OPTIONAL_DOWNLOADER_CLASS = "source.filebased.downloader.class";
 
   /**
+   * Configuration properties used by the LimitingExtractorDecorator
+   */
+  public static final String LIMITER_REPORT_KEY_LIST = "limiter.report.key.list";
```

Comment:
`ConfigurationKeys` is intended for publicly settable configurations. Please put this constant somewhere else (e.g. in the LimiterExtractor)